starboard/shared/uwp/extended_resources_manager.cc

```diff
@@ -221,8 +257,25 @@ bool ExtendedResourcesManager::GetD3D12ObjectsInternal() {
     }
     SB_DCHECK(d3d12queue_);
   }
+  if (!d3d12FrameBuffersHeap_) {
+    D3D12_HEAP_DESC heap_desc;
+    heap_desc.SizeInBytes = kFrameBuffersPoolMemorySize;
+    heap_desc.Properties.Type = D3D12_HEAP_TYPE_DEFAULT;
+    heap_desc.Properties.CPUPageProperty = D3D12_CPU_PAGE_PROPERTY_UNKNOWN;
+    heap_desc.Properties.MemoryPoolPreference = D3D12_MEMORY_POOL_UNKNOWN;
+    heap_desc.Properties.CreationNodeMask = 0;
+    heap_desc.Properties.VisibleNodeMask = 0;
+    heap_desc.Alignment = D3D12_DEFAULT_RESOURCE_PLACEMENT_ALIGNMENT;
+    heap_desc.Flags = D3D12_HEAP_FLAG_NONE;
+
+    if (FAILED(d3d12device_->CreateHeap(
+            &heap_desc, IID_PPV_ARGS(&d3d12FrameBuffersHeap_)))) {
+      SB_LOG(WARNING) << "Failed to create d3d12 buffer.";
+      return false;
+    }
```

Comment:
Might want to add a SB_DCHECK after this if-block to keep consistent with d3d12device_ and d3d12queue_

Reply:
Done